package.json

```diff
@@ -170,7 +162,8 @@
     "react-popper": "^2.2.3",
     "@popperjs/core": "^2.4.0",
     "react-select": "^3.1.0",
-    "react-transition-group": "~2.3.1"
+    "react-transition-group": "~2.3.1",
```

Comment:
We can remove `react-transition-group` from package.json, it's not used in scripts.